src/workerd/io/worker-fs.c++

```diff
@@ -1755,15 +1755,33 @@ class DevRandomFile final: public File, public kj::EnableAddRefToThis<DevRandomF
 void writeStdio(jsg::Lock& js, VirtualFileSystem::Stdio type, kj::ArrayPtr<const kj::byte> bytes) {
   auto chars = bytes.asChars();
   size_t endPos = chars.size();
-  if (chars[endPos - 1] == '\n') endPos--;
+  if (endPos > 0 && chars[endPos - 1] == '\n') endPos--;
+
+  // KJ_LOG(WARNING, "writeStdio called", "content:", chars);
 
   KJ_IF_SOME(console, js.global().get(js, "console"_kj).tryCast<jsg::JsObject>()) {
-    auto method = console.get(js, type == VirtualFileSystem::Stdio::OUT ? "log"_kj : "error"_kj);
+    auto method = console.get(js, "log"_kj);
```

Comment:
If `console.log` is not a function it might be a good idea for us to log a warning... only possible when there's an IoContext but still useful.